src/Feature.jsx

```diff
@@ -0,0 +1,88 @@
+// @flow
```

Comment:
How will this integrate with Optimizely's React component?

Reply:
Integration example - https://github.com/heydoctor/web-app/compare/feat/optimizely-integration...feat/optimizely-integration-react-scientist